src/markerclusterer.js

```diff
@@ -763,18 +757,10 @@ MarkerClusterer.prototype.resetViewport = function(opt_hide) {
  *
  */
 MarkerClusterer.prototype.repaint = function() {
-    var oldClusters = this.clusters_.slice();
     this.clusters_.length = 0;
     this.resetViewport();
     this.redraw();
 
-    // Remove the old clusters.
```

Comment:
This was done twice at repaint and resetViewport, moved the code to resetViewport